src/module/sheets/TwodsixActorSheet.ts

```diff
@@ -56,8 +56,14 @@ export class TwodsixActorSheet extends AbstractTwodsixActorSheet {
     super.activateListeners(html);
 
     // Rollable abilities. Really should be in base class, but that will have to wait for issue 86
+
     html.find('.rollable').on('click', (this._onRoll.bind(this)));
```

Comment:
Here I kept the old way of making a throw. So that it's possible to compare to the old way